configure.ac

```diff
@@ -363,7 +363,7 @@ AC_SUBST(LIBRT)
 AC_CHECK_LIB(m, sqrtf)
 
 # for BeOS - old network stack don't have those libs ( move it in the case switch ?)
-AC_CHECK_LIB([socket], [socket])
+AC_CHECK_LIB([network], [socket])
```

Comment:
same here; I'm not at all literate in configure. Should this be an additional test instead of a change?

Reply:
Fixed. Retest build. Works fine.